src/environmentd/src/telemetry.rs

```diff
@@ -167,7 +166,10 @@ async fn report_loop(
                         'active_subscribes', {active_subscribes}
                     )",
                 )).await?;
-                let row = rows.into_element();
+
```

Comment:
Pretty sure there's a good reason for this change, but isn't the new code exactly what `into_element` does?

Reply:
> but isn't the new code exactly what into_element does?

It is! But `into_element()` is part of the `trait CollectionExt` which is only implemented for types that implement `IntoIterator` which `RowIterator` does not.